### PR TITLE
Bug Fixes

### PR DESCRIPTION
**Pull request by alvinnguyen116**
Fixed: off by one index error, evaluation email link points to correct heroku, tests, readme

### CODE REVIEW

**Comment by Divineechidume** [comment]
Hi An I know it's after the deadline but I made the test testing the sending a tutor an email.

File: app/mailers/tutor_mailer.rb [hunk 1]
```diff
@@ -6,6 +6,7 @@ def invite_student(tutor_id, tutee_id, tutor_message, request_id, eval_id)
 		@request = Request.find_by_id(request_id)
 		@tutor_message = tutor_message
 		@eval = Evaluation.find_by_id(eval_id)
+		puts(tutor_id, tutee_id)
```
Comment (an-ju) [inline app/mailers/tutor_mailer.rb]: Please remove this line.

File: app/views/tutors/find_students.html.erb [hunk 2]
```diff
@@ -66,17 +66,23 @@
...
                                     aria-labelledby="<%=true_class %>-tab">
                                     <% puts '~~~~~~~~~~~~~~~' %>
-                                    <% puts Course.where(name: true_class) %>
-                                    <% course_obj = Course.where(name: true_class).first %>
-                                    <% requests_obj = Request.where(course_id: course_obj.course_num).order(created_at: :asc)%>
+                                    <% x = Course.where(name: true_class, semester: Course.current_semester).first %>
+                                                                        <% puts 'CLASS: ' + true_class %>
+
+                                    <% x.attributes.each do |attr_name, attr_value| %>
+                                    <% puts attr_name %>
```
Comment (an-ju) [inline app/views/tutors/find_students.html.erb]: Why do you have `puts` in the erb file?  Please remove it if it's for debugging.  Please use `<%= %>` if you want to show it in the page.

**Comment by an-ju** [comment]
@Divineechidume There are some minor changes required.  Thanks for the quick bug fix.

**Pull request merged**